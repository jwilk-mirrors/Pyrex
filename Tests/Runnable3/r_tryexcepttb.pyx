def test():
	try:
		raise ValueError("Too many eels in hovercraft")
	except ValueError, e, t:
		print "Exception:", e
		print "Traceback:", t
