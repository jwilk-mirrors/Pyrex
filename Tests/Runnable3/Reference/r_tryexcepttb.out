Exception: Too many eels in hovercraft
Traceback type: <type 'traceback'>
